MFVNeutralino/plugins/MovedTracksTreer.cc

```diff
@@ -277,18 +290,17 @@ void MFVMovedTracksTreer::analyze(const edm::Event& event, const edm::EventSetup
           ce << "--this isn't a valid vertex number (max " << nt.vertices().n() << ")\n";
         throw ce;
       }
+      */
       const size_t iv = nt.vertices().n() - 1;
       assert(iv < 255);
-      nt.tracks().set_which_sv(m.i(), iv);
+      nt.tracks().set_which_sv(99, iv); //FIXME
```

Comment:
It would be good to clean up the commented block (at least leave a comment for why it is commented / what it is for, or better yet, have it part of a debug or verbose option. It would also be good to resolve the FIXME here, where m.i() was replaced with 99. If this is an intended change, let's add a comment for what 99 is rather than leaving the magic number in place.